2020/02.md

```diff
@@ -83,6 +83,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
     |   | 0     | 15m     | [Strict builtin functions](https://github.com/Jack-Works/proposal-strict-built-in-functions) for Stage 1 | Mark S. Miller  (by Jack-Works) |
     |   | 0     | 30m     | [Legacy reflection features for functions in JavaScript](https://github.com/claudepache/es-legacy-function-reflection) for Stage 1 | Mark S. Miller  (by Claude Pache) |
     |   | 0     | 30m     | [function `thisArgumentExpected` property](https://github.com/hax/proposal-function-this) for Stage 1 | 贺师俊 (HE Shi-Jun) |
+    |   | 0     | 60m     | Extensions for Stage 1 | 贺师俊 (HE Shi-Jun) |
```

Comment:
Will there be a link provided?

Reply:
Was mentioned above that it will be included later

@hax do you have a TL;DR about what this proposal is focused on?